src/components/DateRangePicker.js

```diff
@@ -547,7 +582,7 @@ const DateRangePicker = React.createClass({
         fontSize: StyleConstants.FontSizes.MEDIUM,
         paddingLeft: isLargeOrMediumWindowSize ? 10 : 0,
         paddingTop: 10,
-        maxWidth: 250,
+        maxWidth: window.innerWidth > 450 && 250,
```

Comment:
I'm with leaving it the way it is.  I just thought we just double check cause I wasn't sure how the browser would feel about assigning a boolean to property like that.  Thanks for double checking.